srcpkgs/wayland/template

```diff
@@ -1,18 +1,24 @@
 # Template file for 'wayland'
 pkgname=wayland
-version=1.21.0
+version=1.22.0
 revision=1
 build_style=meson
-# "Tests must not be built with NDEBUG defined, they rely on assert()."
-configure_args="-Ddocumentation=false -Db_ndebug=false"
+configure_args="-Ddocumentation=false"
 hostmakedepends="flex pkg-config"
 makedepends="expat-devel libffi-devel libfl-devel libxml2-devel"
 short_desc="Core Wayland window system code and protocol"
 maintainer="Érico Nogueira <ericonr@disroot.org>"
 license="MIT"
 homepage="https://wayland.freedesktop.org/"
 distfiles="https://gitlab.freedesktop.org/wayland/wayland/-/releases/${version}/downloads/wayland-${version}.tar.xz"
-checksum=6dc64d7fc16837a693a51cfdb2e568db538bfdc9f457d4656285bb9594ef11ac
+checksum=1540af1ea698a471c2d8e9d288332c7e0fd360c8f1d12936ebb7e7cbc2425842
+
+if [ "$XBPS_CHECK_PKGS" ]; then
+	# "Tests must not be built with NDEBUG defined, they rely on assert()."
+	configure_args+=" -Dtests=true -Db_ndebug=false"
```

Comment:
Having `b_ndebug` disabled when building with tests and disabled otherwise doesn't sit entirely right with me, since it may affect the non-test code too. Thoughts?

Reply:
let's keep it as it was